### DISCUSSION TITLE
Having multiple config files

### DISCUSSION THREAD

**Discussion by kipusoep** [renovatebot/renovate#14819] [Request Help] [answered]
As far as I can tell it's not possible to have multiple config files, is it?
The thing is; I'd like to check every week for 3rd party package updates (only minor/patch updates and `stabilityDays` set to 7).
Our internal packages are also checked, but without the `stabilityDays` setting.
But whenever we update certain packages by pushing to it's repository, a pipeline in repositories using that package is triggered and I'd like to run renovate for only our internal packages.
So my question is, if it's not possible to have multiple config files, is it possible to adjust the `packageRules` somehow to disable checking 3rd party updates? Or is there some other way to make this work?

**Comment by rarkins** [comment, marked as answer]
It's not possible to have multiple config files.

I don't understand your use case either though. Please look at `schedule` and `updateNotScheduled` config options and if you can explain why those are not sufficient then I can maybe understand better.

**Reply by kipusoep** [reply]
Hmm adding `"extends": ["schedule:weekends"]` to the 3rd party package rule might work for me. I'm going to test that and report back.

**Reply by kipusoep** [reply]
Yes, works for me! Thanks :-)
